perl/README.md

```diff
@@ -0,0 +1,26 @@
+# Bystro Annotator
+
+## Install
+
+```bash
+# system install
+cpanm Dist::Zilla
+dzil install
```

Comment:
So I can't get this to quite work. Also tried cpanm the built tarball, same error:

```
2023-09-16 22:50:38 [info] ref: Beginning building from /home/alexkotlar/bystro2/perl/.build/u4e7cswOD8/t/tracks/gene/db/raw/ref/chrM.fa.gz
pigz: skipping: /home/alexkotlar/bystro2/perl/.build/u4e7cswOD8/t/tracks/gene/db/raw/ref/chrM.fa.gz does not exist
```

perl/t/tracks/gene/db/raw/ref/chrM.fa.gz definitely exists, but seems like it isn't getting packaged?

Reply:
You're right. Binary files are excluded because this caused the packaging to break. I can update the README with the approach used for building the docker container, which works for installing it locally, too. I suggest making this a goal for automated testing. Right now there are two perl packages that need to be installed outside of the normal cpanm approach.

Reply:
Would making it just a .fa fix that?

Reply:
And yes, would love to get the updated README so that users / I can understand how to install this.

Reply:
README.md was updated with latest approach to building.